src/language.cc

```diff
@@ -14,9 +14,13 @@ using namespace Napi;
 const TSLanguage *UnwrapLanguage(const Napi::Value &value) {
   Env env = value.Env();
 
-  const TSLanguage *language = static_cast<const TSLanguage *>(
-    GetInternalFieldPointer(value)
-  );
+  const TSLanguage *language
```

Comment:
@maxbrunsfeld this is the only thing I'm unsure about. I remember having issues loading my language when I originally made this branch, so I added to that language (somewhere, I forget) a special property `_language` which is a `Napi::External` with a pointer to the `TSLanguage`. I can probably remove this change, tests should still pass, but I do think there is probably some untested issue that I need to recover, unrelated to merging with master. Thoughts on me removing this change in this PR and deferring finding what the issue was and solving it to another PR?

Reply:
I need to refresh on how much of this should be generated, but apparently in my fork of `tree-sitter-sql` I modified the node binding module export to export the `TSLanguage` as a `Napi::External` for loading here. I did that [here](https://github.com/m-novikov/tree-sitter-sql/compare/main...MichaelBelousov:tree-sitter-sql:use-napi-instead-of-nan.)

Reply:
Is this what @maxbrunsfeld was talking about in https://github.com/tree-sitter/node-tree-sitter/pull/52#issue-518154583?

> Unfortunately, we can't achieve true ABI-stability all in one fell swoop. This module has two binary interfaces that currently depend on Nan / V8 directly:
>
> * Retrieving the TSLanguage * pointers out of Language objects from language modules. Currently, the Node.js binding code that the tree-sitter CLI generates for individual languages are reliant on nan and the V8 ABI 😞 .

Reply:
Almost definitely that's it. Well I had a workaround clearly but it's worth discussing.

Reply:
What about this comment: https://github.com/tree-sitter/node-tree-sitter/pull/52#issuecomment-779483061 ?